Included uci webauth service to make sign in easier for people with u…

Guvn0 opened the pull request:
…ci accounts. In order to be able to use webauth with this site it needs to be hosted on a ".uci.edu" domain, runapp.sh is modified with "export FLASK_RUN_HOST=shpe.uci.edu" in order to account for this, however it seems that to run server locally and work correctly you will need to modify local hosts file: in Linux systems this is found at "/etc/hosts"; in Windows systems it is found at "C:\Windows\System32\Drivers\etc\hosts"; not sure about Mac systems, but it maybe be same as Linux. Inside the hosts file you should see "127.0.0.1	 localhost" if you change this to "127.0.0.1	shpe.uci.edu#localhost" the sign in should work with no issues. Note: shpe in shpe.uci.edu can be replaced with whatever else you choose but I thought it looked more official than anything else I could come up with so I kept it.

Guvn0 committed:
Merge branch 'master' into uci-webauth

Guvn0 committed:
Merge branch 'master' into uci-webauth

josueal1 reviewed (commented):
Holding off approval until others are merged, then conflicts or branch deletions will follow @guilleeh @Guvn0

josueal1 committed:
Merge branch 'master' into uci-webauth

josueal1 committed:
style(app.py): swap single quotes with double quotes and add more linter-recommended spacing
Motivation: linters recommend double quotes because they prevent any string issues

josueal1 committed:
style(app.py): Reorder the import lines in std, 3rd party, local

Motivation: To set an example for future developers to follow PEP8 import practices

josueal1 committed:
style(app.py): Wrap long lines and comment unused variables

josueal1 committed:
WIP(app.py): Delete temporary lint-log.txt file - Work In Progress
